dbt_subprojects/dex/macros/models/_project/valantis_compatible_trades.sql

```diff
@@ -0,0 +1,54 @@
+{% macro valantis_compatible_hot_trades(
+    blockchain = null
+    , project = null
+    , version = null
+    , HOT_evt_Swap = null
+    , Pair_evt_Swap = null
+    , Factory_evt_PoolCreated = null
+    )
+%}
+WITH dexs AS
+(
+    SELECT distinct t.evt_block_number AS block_number
+        , t.evt_block_time AS block_time
+        , amountOut AS token_bought_amount_raw
+        , amountIn AS token_sold_amount_raw
+        , CASE WHEN isZeroToOne THEN f.token1 ELSE f.token0 END AS token_bought_address
+        , CASE WHEN isZeroToOne THEN f.token0 ELSE f.token1 END AS token_sold_address
+        , t.sender AS taker
+        , coalesce(h.contract_address, t.contract_address) AS maker -- HOT for HOTSwaps (solver orders), SovereignPool for AMM swaps (permissionless)
+        , t.contract_address AS project_contract_address
+        , t.evt_tx_hash AS tx_hash
+        , t.evt_index AS evt_index
+    FROM {{ source('erc20_' + blockchain, 'evt_transfer') }} AS e
```

Comment:
what is the intention of reading from this table? from the select, it looks like you can simply select from `{{ Pair_evt_Swap }}` and then join to the other two decoded tables

Reply:
this could play a part in your CI failure, as it makes the query much heavier to read from

Reply:
the error means the cluster is crashing during the build

Reply:
ohhh, i see! i will try to get rid of it, thanks 🙂